metadata-ingestion/src/datahub/utilities/urns/urn_iter.py

```diff
@@ -117,17 +117,19 @@ def _modify_at_path(
         if isinstance(path[0], int):
             assert isinstance(model, list)
             model[path[0]] = new_value
-        elif isinstance(model, MetadataChangeProposalWrapper):
-            setattr(model, path[0], new_value)
-        else:
-            assert isinstance(model, DictWrapper)
+        elif isinstance(model, DictWrapper):
             model._inner_dict[path[0]] = new_value
+        else:
+            # isinstance(model, MetadataChangeProposalWrapper)
```

Comment:
Remove these comments? Or maybe just say, this is an MPC / MCPC (assume that's why we're catching an else case?)

Reply:
done